components/EditContentMenuItem.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import MenuItem from "@mui/material/MenuItem";
+import Modal from "@mui/material/Modal";
+import '@/app/css/Affirmation.css';
+
+interface EditContentMenuItemProps {
+  id: number;
+  initialText: string;
+  updateContent: (id: number, updatedText: string) => void;
```

Comment:
This should also start with "on", I think

Reply:
Done